ci/generate-markdown/generate-packaging.sh

```diff
@@ -199,7 +199,7 @@ cc_binary(
 
 | Library | Minimum version | Description |
 | ------- | --------------: | ----------- |
-| [Abseil][abseil-gh] | 20200923, Patch 3 | Abseil C++ common library |
```

Comment:
We have at least one build that tests with 20200923.  I think we should not drop support for the older version, if we do, we likely have to drop support for older gRPC versions too.

Reply:
The issue is that this version of Abseil renames their old `.so` files and adds proper `pkg-config` support. So ideally we should start using their pkg-config files, as I've done in this version of the PR, but doing that makes it so our `Makefile` builds will no longer work w/ previous releases of Abseil.

We could also not using Abseil's `pkg-config` files that are added in this new release. But doing that would require that we change our CMake function to understand Abseil's new `.so` naming.

Reply:
I think I follow how we end up here.

I am trying to think about this from the perspective of customers reading this document, maybe for the first time.  They read `Abseil >= 20210324` and also read `gRPC >= 1.35` which supposedly works with (requires?) `Abseil == 20200923`.  That seems complicated.  Which one is it?

Anyway, feel free to ignore this.  The thing that @devbww points out seems more important.